src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryPipeStatus.java

```diff
@@ -40,6 +40,10 @@
   AtomicLong cleanerRestartCount;               // how many times the cleaner restarted
   static final String CLEANER_RESTART_COUNT     = "cleaner_restart_count";
 
+  // Memory usage
+  AtomicLong memoryUsage;               // how many times the cleaner restarted
```

Comment:
what do you mean?
`how many times the cleaner restarted`

Reply:
Sorry copy&paste error